app/components/candidate_interface/application_status_tag_component.html.erb

```diff
@@ -40,7 +40,7 @@
     </p>
   <% elsif  @application_choice.offer_deferred? %>
     <p class="govuk-body govuk-!-margin-top-2">
-      Your training will now start in <%= (@application_choice.current_course_option.course.start_date + 1.year).to_fs(:month_and_year) %>.
+      Your training will now start in <%= @application_choice.current_course.deferred_start_date.to_fs(:month_and_year) %>.
```

Comment:
current_course_option.course and current_course are the same thing I think https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/models/application_choice.rb#L20

Reply:
Ah, I'm thinking of confirmed deferrals. This is good.

Reply:
When you confirm a deferral the application choice status changes to recruited so this code won't be called then